grid.css

```diff
@@ -2,27 +2,22 @@
 heading1, heading2{
 	width: 100%;
 	display: grid;
-  grid-gap: 2%;	
-	grid-template-columns: 12% 12% 12% 12% 12% 12% 12%;
+    grid-gap: 2%;	
+	grid-template-columns: 8% 25% 12% 14% 16% 8% 8%;
 }
 
 /* style for the headings section */
 heading1{
-	background-color: #0f0;
-  border-bottom: 1px solid #aaa;
+    border-bottom: 1px solid #aaa;
```

Comment:
I removed the green background for the headings section as I did need to visualise it anymore.